db/sql/sql.go

```diff
@@ -505,14 +505,18 @@ func makeColumn(tableName string, property schema.Property) string {
 }
 
 // MakeColumns generates an array that has Gohan style colmun names
-func MakeColumns(s *schema.Schema, tableName string, join bool) []string {
+func MakeColumns(s *schema.Schema, tableName string, join bool, aliasTableName string) []string {
 	var cols []string
 	manager := schema.GetManager()
 	for _, property := range s.Properties {
-		cols = append(cols, makeColumn(tableName, property)+" as "+quote(makeColumnID(tableName, property)))
+		if aliasTableName == "" {
+			aliasTableName = tableName
+		}
+		cols = append(cols, makeColumn(aliasTableName, property)+" as "+quote(makeColumnID(tableName, property)))
```

Comment:
I don't understand this part.  

quote(makeColumnID(tableName, property)

I guess there would be case in that some properties get shadowed, right?
quote(makeColumnID(aliasTableName, property) works?

In this case, we can omit aliasTableName too